src/core/react-query/group/mutations.ts

```diff
@@ -0,0 +1,56 @@
+import { useMutation } from '@tanstack/react-query';
+
+import toast from '@/components/Toast';
+import { axios } from '@/core/axios';
+import { invalidateQueries } from '@/core/react-query/queryClient';
+
+import type {
+  CreateGroupRequestType,
+  MoveSeriesGroupRequestType,
+  PatchGroupRequestType,
+} from '@/core/react-query/group/types';
+
+/**
+ * This file probably needs more work on query invalidation.
+ * Currently, it's naïve and only invalidates the current series query.
+ * It should probably also invalidate the cache for:
+ *  * Any series belonging to the "original" groups of a series.
+ */
```

Comment:
@fearnlj01 let's discuss on discord 😔 
It's a bit tricky because we already invalidate on SeriesUpdated event from signalr. So these invalidations might not be required at all.